docs/getting_started.rst

```diff
@@ -295,7 +295,6 @@ to a working directory:
 
 .. code-block:: shell
 
-  mkdir simulator-example
   cp -rf NVFlare/examples/hello-world/hello-pt simulator-example/
```

Comment:
@hwpang to ensure the behavior is always consistent, we probably need to add a `rm -rf simulator-example` in front of this command to achieve what we want

Reply:
@YuanTingHsieh That makes sense, thanks! Updated!